### PR TITLE
#162633468 Implements Like or dislike articles by a user

### PR DESCRIPTION
**Pull request by mubstimor**
**What does this PR do?**

Implements Like or dislike articles by a user

**Description of Task to be completed?**

- Track whether logged in user's likes or dislikes an article.

**How should this be manually tested?**

- Run `npm install` to install app dependencies.
- Run `npm test` to test the application.
- Run `npm start` to run the application.

**What are the relevant pivotal tracker stories?**

[#162633468](https://www.pivotaltracker.com/story/show/162633468)

### CODE REVIEW

**Comment by jeanjoe** [comment]
@mubstimor Please remove the coverage folder

**Comment by mubstimor** [comment]
> @mubstimor Please remove the coverage folder

removed

**Comment by NabunyaLilian** [comment]
LGTM

**Comment by ruganda** [comment]
Good work, @mubstimor, please resolve the conflicts and remove the console logs in your code.

File: src/actions/articleActions.js [hunk 1]
```diff
@@ -100,14 +100,14 @@ export const deleteArticle = slug => ( dispatch ) => {
...
-      dispatch( editArticleSuccess( response.data ) );
-    } )
-    .catch( ( err ) => {
-      console.log( 'error' );
-      dispatch( editArticleFailure( err.response ) );
-    } );
+export const updateArticle = (slug, articleToEdit) => (dispatch) => {
+  api.updateResource(`${articleUrl}${slug}/`, articleToEdit)
+    .then((response) => {
+      console.log("created");
```
Comment (ruganda) [inline src/actions/articleActions.js]: Please remove this console logs

**Comment by mubstimor** [comment]
> Good work, @mubstimor, please resolve the conflicts and remove the console logs in your code.

have resolved the conflicts.

**Comment by jeanjoe** [comment]
Please resolve your conflicts

**Comment by jeanjoe** [comment]
@mubstimor Please add back the liks and dislikes count to the single article even when you are not authenticated

**Pull request merged**